### PR TITLE
refactor: update messageType values

### PR DESCRIPTION
**Pull request by yeul**

### CODE REVIEW

File: src/server/locale/US/CONTEXTUAL_COMPONENTS/BUTTONS/mutations/index.js [hunk 1]
```diff
@@ -18,35 +18,43 @@ export default function getMutations(id, type) {
...
         // Short Term Non-qualifying
         case 'ST_NQ':
+        case 'PLST_NQ':
             return shortTermSingleNQ[type];
         // Generic Messages
         // Generic Pay Later message
         case 'GENERIC_PL':
             return genericPayLater[type];
         // // Generic Pay Later - Buy now, Pay Later message
-        case 'GENERIC_PL_BNPL':
```
Comment (Seavenly) [inline src/server/locale/US/CONTEXTUAL_COMPONENTS/BUTTONS/mutations/index.js:49]: Is this removal intentional? It seems like in every other case we're just adding, so wanted to double check.

File: src/server/locale/US/CONTEXTUAL_COMPONENTS/MARKS/mutations/index.js [hunk 2]
```diff
@@ -18,35 +18,43 @@ export default function getMutations(id, type) {
...
         // Short Term Non-qualifying
         case 'ST_NQ':
+        case 'PLST_NQ':
             return shortTermSingleNQ[type];
         // Generic Messages
         // Generic Pay Later message
         case 'GENERIC_PL':
             return genericPayLater[type];
         // // Generic Pay Later - Buy now, Pay Later message
-        case 'GENERIC_PL_BNPL':
```
Comment (Seavenly) [inline src/server/locale/US/CONTEXTUAL_COMPONENTS/MARKS/mutations/index.js:49]: Same comment as above.

**Review by Seavenly — APPROVED** [review]

File: src/server/locale/US/CONTEXTUAL_COMPONENTS/BUTTONS/mutations/index.js (hunk 1 above)
Comment (yeul) [reply src/server/locale/US/CONTEXTUAL_COMPONENTS/BUTTONS/mutations/index.js:49]: Good catch, fixed that up for buttons and marks. 👍

**Review by Seavenly — APPROVED** [review]
